cmd/mt-store-cat/main.go

```diff
@@ -182,6 +184,7 @@ func main() {
 	if err != nil {
 		log.Fatalf("failed to initialize cassandra. %s", err.Error())
 	}
+	idx := cassandra_idx.New(idxConfig)
```

Comment:
doesn't this `idxConfig` need to be configured? just like the `storeConfig` is configured with all the cli parameters

Reply:
Fixed in latest commit